docs/quick-start.ipynb

```diff
@@ -29,9 +27,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ds = xr.tutorial.load_dataset(\"eraint_uvz\")\n",
+    "ds = xr.tutorial.load_dataset(\"eraint_uvz\").astype(\"float32\")\n",
```

Comment:
do you approve of this fix in the quickstart or better take another sample dataset?

Reply:
I think this is okay for now. Could you mention this specific commit in issue #168 so that we do not forget to revert it.
Other datasets in the tutorial are often also offset/scale compressed leading to float64. The best alternative might be "ROMS_example", but since the coordinates are quite complex it might cause some new issues.